.renovaterc.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": [
+    ":gitSignOff",
+    "group:monorepos",
+    "group:recommended",
+    ":pinAllExceptPeerDependencies",
+    "helpers:disableTypesNodeMajor"
```

Comment:
IIUC this helper makes sure renovatebot doesn't update `@types/node` to a new semver major release right?

Reply:
Yes, exactly 👍

Reply:
See https://renovatebot.com/docs/presets-helpers/#helpersdisabletypesnodemajor

```json
{
  "packageRules": [
    {
      "packageNames": [
        "@types/node"
      ],
      "major": {
        "enabled": false
      }
    }
  ]
}
```